### PR TITLE
Mobile Styles 835pm

### PR DESCRIPTION
**Pull request by robertobuso**
Fixed padding issues re navbar/headers in all views

### CODE REVIEW

File: static/ca_bydate.js [hunk 1]
```diff
@@ -173,9 +173,11 @@ function redoFilter() {
   table.clearFilter();
   if (countySel.value) {
     table.addFilter("county", "=", countySel.value);
+    setSubHeader = document.getElementById("sub-header").innerHTML = countySel.value;
```
Comment (dsjoerg) [inline static/ca_bydate.js]: looks like this is unused. i'm gonna remove it from the version it put into master. let me know if i'm missing something.

File: static/ca_bydate.js [hunk 2]
```diff
@@ -173,9 +173,11 @@ function redoFilter() {
...
   }
   if (locationTypeSel.value) {
     table.addFilter("location_type", "=", locationTypeSel.value);
+    setSubHeader = document.getElementById("sub-header").innerHTML = locationTypeSel.value;
```
Comment (dsjoerg) [inline static/ca_bydate.js]: this too

Comment (robertobuso) [reply static/ca_bydate.js]: We use these lines of code to show the name of the specific county or venue that the user selects as a header above the chart of both of the chart pages

File: static/ca_bydate.js (hunk 1 above)
Comment (robertobuso) [reply static/ca_bydate.js]: See comment below

**Pull request merged**